docs/zh-cn/1.3.0/user_doc/cluster-deployment.md

```diff
@@ -210,40 +211,21 @@ mysql -uroot -p
     # 数据库密码, 如果有特殊字符，请使用\转义，需要修改为上面设置的{passowrd}具体值
     passowrd="xxx"
 
-    #将DS安装到哪个目录，如: /opt/soft/dolphinscheduler，不同于现在的目录
-    installPath="/opt/soft/dolphinscheduler"
-
-    #使用哪个用户部署，使用1.3小节创建的用户
-    deployUser="dolphinscheduler"
-
     #Zookeeper地址，单机本机是localhost:2181，记得把2181端口带上
     zkQuorum="localhost:2181"
```

Comment:
cluster is "192.168.xx.xx:2181,192.168.xx.xx:2181,192.168.xx.xx:2181"

Reply:
I have modified.